### PR TITLE
redoing the same question ealier request closed

### PR DESCRIPTION
**Pull request by garunanurag**
while copying i made changes to some other users answer therefore  earlier pull request was closed hope its fine this time

### CODE REVIEW

**Comment by garunanurag** [comment]
please review it

**Comment by dakshkhetan** [comment]
@garunanurag You have made changes to root's `index.html`, please remove those changes.
Else everything is fine.

**Comment by garunanurag** [comment]
Ok  doing it now

**Comment by garunanurag** [comment]
i hope its fine now

File: javascript/answers/Anurag-Pandey-q36.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content=" Does JavaScript support automatic type conversion?">
```
Comment (shivamyth) [inline javascript/answers/Anurag-Pandey-q36.html]: change content in meta tag

Comment (shivamyth) [inline javascript/answers/Anurag-Pandey-q36.html]: change content in the meta tag

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: javascript/answers/Anurag-Pandey-q36.html (hunk 1 above)
Comment (garunanurag) [reply javascript/answers/Anurag-Pandey-q36.html]: is it fine now

File: javascript/answers/Anurag-Pandey-q36.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="JavScript interview questions">
```
Comment (shivamyth) [inline javascript/answers/Anurag-Pandey-q36.html]: You have to write your question title there, i.e.  "What is eventloop in javascript?"

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: javascript/answers/Anurag-Pandey-q36.html (hunk 2 above)
Comment (garunanurag) [reply javascript/answers/Anurag-Pandey-q36.html]: ok
done

**Review by shivamyth — APPROVED** [review]

**Pull request merged**